samples/eshop/services/basket.bicep

```diff
@@ -54,19 +45,18 @@ resource basket 'Applications.Core/containers@2023-10-01-preview' = {
   properties: {
     application: application
     container: {
-      image: 'ghcr.io/radius-project/samples/eshop/basket.api:${TAG}'
+      image: '${imageRegistry}/basket.api:${imageTag}'
       env: {
         ASPNETCORE_ENVIRONMENT: 'Development'
         ASPNETCORE_URLS: 'http://0.0.0.0:80'
-        ApplicationInsights__InstrumentationKey: APPLICATION_INSIGHTS_KEY
         UseLoadTest: 'False'
         PATH_BASE: '/basket-api'
-        OrchestratorType: ORCHESTRATOR_TYPE
+        ORCHESTRATOR_TYPE: 'K8S'
```

Comment:
Why did we need to change this?

Reply:
We used to have this be a parameter that could be set to only the value 'K8S'. Since we don't support anything else we just hard coded it. Goal is to simplify the parameter list